subprojects/docs/src/docs/userguide/application_plugin.adoc

```diff
@@ -138,8 +138,12 @@ _Depends on_: `jar`, `startScripts`
 +
 Creates a full distribution TAR archive including runtime libraries and OS specific scripts.
 
+[[sec:application_extension]]
+== Application extension
+
+The application plugin adds an extension to the project, which you can use to configure its behaviour. See the link:{groovyDslPath}/org.gradle.api.plugins.JavaApplication.html[JavaApplication] DSL documentation.
 
 [[sec:application_convention_properties]]
-== Convention properties
+== Deprecated convention properties
```

Comment:
I would prefer:

> == Convention properties (deprecated)

Your proposal could imply that there are non-deprecated convention properties.